Assets/HotmaxEditorVR/builder/uiClasses/editorModeUIController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections;
```

Comment:
I'm not sure about the naming classes "Controller", only because we aren't really following an MVC model, so people might get confused with the nomenclature. Also because we've began naming things Control quite often, this could get confusing.

I think "Manager" might be better.
Same with PropController --> Prop(s)Manager.

Reply:
I also have questions with how functionality is split between this class and editorModeUI. It seems like there are some "controller"-esque behavior in the editorModeUI class for instance.

Reply:
got it, makes sense